test/runnable/testassert.d

```diff
@@ -37,8 +37,53 @@ void test8765()
         assert(file && file == "runnable/testassert.d");
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=20114
+void test20114()
```

Comment:
Test should definitely include non-call side effects, e.g. `++i` where `i` is an `int`.

Reply:
Added